Campus Density/HomeFeedbackQuestions/ReviewQuestion.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ReviewQuestion.swift
+//  Campus Density
+//
+//  Created by Mihikaa Goenka on 07/03/21.
+//  Copyright © 2021 Cornell DTI. All rights reserved.
+//
+
+import UIKit
+
+protocol ReviewQuestionDelegate: class {
+    func reviewWasChanged(review: Int)
+}
+
+class ReviewQuestion: HomeFeedbackQuestion {
+    weak var delegate: ReviewQuestionDelegate?
+    let colors: [UIColor] = [.lightTeal, .wheat, .peach, .orangeyRed] // From CurrentDensityCell
+    let noLikeLabel = UILabel()
+    let likeLabel = UILabel()
+    var colorPills: [UIButton] = []
+    var radioButtons: [UIButton] = []
+
+    init() {
+        super.init(subtitle: "3. How do you like Flux overall?")
```

Comment:
We need to make this question necessary too. Right now it's possible to just skip it.